### PR TITLE
iOS fixes

### PR DESCRIPTION
**Pull request by kassemitani**
iOS fixes

### CODE REVIEW

**Review by FokkeZB — CHANGES_REQUESTED** [review]
Thanks for the PR! Could you stick with 2 spaces for the XML so that I can see the actual changes you made?

**Comment by kassemitani** [comment]
Now its okay?

File: tiapp.xml [hunk 1]
```diff
@@ -91,4 +95,4 @@
   <plugins>
     <plugin version="1.0">ti.alloy</plugin>
   </plugins>
-</ti:app>
+</ti:app>
```
Comment (FokkeZB) [inline tiapp.xml:97]: Please make sure there's newline at the EOF.

File: tiapp.xml [hunk 2]
```diff
@@ -24,6 +24,10 @@
 
     <plist>
       <dict>
+		<key>NSLocationAlwaysUsageDescription</key>
```
Comment (FokkeZB) [inline tiapp.xml]: Fix identation

**Review by FokkeZB — CHANGES_REQUESTED** [review]
Almost there. Just some whitespace issues.

Thanks!

**Comment by kassemitani** [comment]
done

**Comment by FokkeZB** [comment]
Thanks @kassemitani - it was missing the OEF newline again, but let's not bother :D

**Pull request merged**